components/subject/RecordAuthorStatus.tsx

```diff
@@ -0,0 +1,65 @@
+import { LabelChip } from '@/common/labels'
+import {
+  NEW_ACCOUNT_MARKER_THRESHOLD_IN_DAYS,
+  YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS,
+} from '@/lib/constants'
+import { ToolsOzoneModerationDefs } from '@atproto/api'
+import {
+  LockClosedIcon,
+  MoonIcon,
+  ShieldExclamationIcon,
+  SunIcon,
+} from '@heroicons/react/24/solid'
+import { differenceInDays, formatDistance } from 'date-fns'
+
+export const RecordAuthorStatus = ({
+  repo,
+}: {
+  repo: ToolsOzoneModerationDefs.RepoView
+}) => {
+  const accountAge = differenceInDays(new Date(repo.indexedAt), new Date())
+  const isNew = accountAge < NEW_ACCOUNT_MARKER_THRESHOLD_IN_DAYS
+  const isYoung = accountAge < YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS
+  const deactivatedAt = repo.deactivatedAt
+    ? formatDistance(new Date(repo.deactivatedAt), new Date(), {
+        addSuffix: true,
+      })
+    : ''
+
+  const isTakendown = !!repo.moderation.subjectStatus?.takendown
+  return (
+    <>
+      {(isNew || isYoung) && (
+        <LabelChip
+          className="dark:bg-slate-700 dark:text-gray-200 rounded-xs"
+          title={`Account was created less than ${NEW_ACCOUNT_MARKER_THRESHOLD_IN_DAYS} days ago`}
```

Comment:
Shouldn't the `YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS` be used instead of `NEW_ACCOUNT_MARKER_THRESHOLD_IN_DAYS` when `isYoung` is true but not `isNew`?